packages/inter-protocol/src/price/fluxAggregator.js

```diff
@@ -63,20 +63,26 @@ const priceDescriptionFromQuote = quote => quote.quoteAmount.value[0];
  * the *Node Operator Aggregation* logic of [Chainlink price
  * feeds](https://blog.chain.link/levels-of-data-aggregation-in-chainlink-price-feeds/).
  *
+ * @param {Baggage} baggage
  * @param {ZCF<ChainlinkConfig & {
  * timer: TimerService,
  * brandIn: Brand<'nat'>,
  * brandOut: Brand<'nat'>,
  * unitAmountIn?: Amount<'nat'>,
  * }>} zcf
- * @param {{
- * marshaller: Marshaller,
- * quoteMint?: ERef<Mint<'set'>>,
- * storageNode: ERef<StorageNode>,
- * }} privateArgs
- * @param {Baggage} baggage
+ * @param {TimerService} timerPresence
```

Comment:
Why timer**Presence**?  We don't usually call out presences as a distinct type.  Would it be sufficient to use timerP to indicate that it's remote? Is there some other `timer` facet to distinguish it from?